node-backend/app/controllers/v1/CoreValueController.js

```diff
@@ -20,11 +25,13 @@ module.exports.create = (req, res) => {
       // Save coreValue in the database
       CoreValue.create(coreValue)
         .then((data) => {
+          logger.info(data);
```

Comment:
Your data should be logged like
-------------------------------------
Executing core value insert
Core value data :  <json data of core value> should be converted to json string
------------------------------------

Same applies to all insert and update functions.